aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -174,6 +174,10 @@ module AptosFramework::Coin {
         amount: u64,
         burn_cap: &BurnCapability<CoinType>,
     ) acquires CoinInfo, CoinStore {
+        if (amount == 0) {
```

Comment:
do we need this? doesn't look like this will cause problem

Reply:
I think we should abort in this case.

Reply:
Yeah abort makes more sense here.